docs/specs/tools/31-ingress-wiki.md

```diff
@@ -0,0 +1,93 @@
+---
+id: ingress-wiki
+sidebar_label: Ingress Wiki
+date: 20240119
+---
+
+# Ingress Wiki Tool
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The **Ingress Wiki Tool** is a [TEv2 text conversion tool](@) that takes Github Wikis that contain [Ingressable Wiki Files](@) and converts them into [curated text](@).
```

Comment:
Are you sure this tool qualifies as a TEv2 text conversion tool? That would mean it has an interpreter profile, one or more predefined interpreters (of which one is the default), a converter profile, and one or more predefined converters, that would all need to be documented (but I don't see the sections here...)